metricflow/query/group_by_item/resolution_dag/dag.py

```diff
@@ -31,9 +31,7 @@ class GroupByItemResolutionDag(MetricFlowDag[GroupByItemResolutionNode]):
 
     def __init__(self, sink_node: ResolutionDagSinkNode) -> None:  # noqa: D
         super().__init__(
-            dag_id=DagId.from_str(
-                IdGeneratorRegistry.for_class(self.__class__).create_id(IdPrefix.GROUP_BY_ITEM_RESOLUTION_DAG.value)
-            ),
+            dag_id=DagId.from_str(PrefixIdGenerator.create_next_id(IdPrefix.GROUP_BY_ITEM_RESOLUTION_DAG.value)),
```

Comment:
Is this right, or should it be `create_next_id(IdPrefix.GROUP_BY_ITEM_RESOLUTION_DAG)` instead?